emc2/core/model.py

```diff
@@ -164,7 +183,7 @@ def load_subcolumns_from_netcdf(self, file_name):
 
 
 class ModelE(Model):
-    def __init__(self, file_path):
+    def __init__(self, file_path, time_range = None):
```

Comment:
For keyword arguments, there should be no space between the variables and the equal signs (see also line 111).